commons-app/src/main/java/cz/cuni/mff/xrg/odcs/commons/app/user/User.java

```diff
@@ -81,6 +94,12 @@
     @OneToOne(mappedBy = "user", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
     private UserNotificationRecord notification;
 
+    @Column(table = "usr_extuser", name = "id_extuser")
+    private String externalIdentifier;
+
+    @Transient
+    private Organization organization;
```

Comment:
This is OK. 

According to explanation from @peterklimo, this is OK as there is no fixed relation between user and org - a user may be part of different organisations during different sessions. So org ID is not a column in user DB table, but the relation is created online, in memory.